bin/rbenv-ctags

```diff
@@ -17,6 +17,16 @@ generate_ctags_in() {
   (cd "$1"; ctags -R --languages=ruby)
 }
 
+generate_c_ctags_in() {
```

Comment:
I think the thing to do here is make `generate_ctags_in` take an optional languages argument and call `--languages="${2:-ruby}"`. Then move the rest of the logic inline.